src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -10,31 +10,25 @@ import { ArtworkSidebarBidAction_artwork } from "__generated__/ArtworkSidebarBid
 
 export interface ArtworkSidebarBidActionProps {
   artwork: ArtworkSidebarBidAction_artwork
-  readonly me?: {
-    readonly bidders: Array<{
-      readonly qualified_for_bidding: boolean
-    }>
-    readonly bidder_status: {
-      readonly active_bid?: {
-        readonly __id: string
-      }
-    }
-  }
 }
 
 export class ArtworkSidebarBidAction extends React.Component<
   ArtworkSidebarBidActionProps
 > {
   render() {
-    const { artwork, me } = this.props
-    const registrationAttempted = me && me.bidders && me.bidders.length > 0
+    const { artwork } = this.props
+    const registrationAttempted = !!artwork.sale.registrationStatus
```

Comment:
Is the `!!` means that the attribute is present and not blank? I always get confused with those double negative things...

Reply:
Use a REPL when in doubt!

```
$ node
> !!{}
true
> !!null
false
```

So a double bang converts truthy/falsy values into absolute booleans.